airflow/api_fastapi/auth/managers/simple/simple_auth_manager.py

```diff
@@ -327,6 +337,21 @@ def _is_authorized(
             return role.order >= allow_get_role.order
         return role.order >= allow_role.order
 
+    @staticmethod
+    def _get_passwords(users: list[dict[str, str]], stream: TextIO) -> tuple[dict[str, str], bool]:
+        # Read passwords from file
+        user_passwords_from_file = json.load(stream)
```

Comment:
Should we handle any non-JSON stream case like `JSONDecodeError` if the file is updated somehow?

Reply:
Exactly, maybe printing on the used class above in case any JSONDecode error is raised similar to the below informing with username and password, we can warn that something is wrong with the file. 
I think it is good to have so it depends on your timing
```
self._print_output(
        f"Password for user '{user['username']}': {passwords[user['username']]}"
)
```

Reply:
I added a `log.error` to make it easier to debug